src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+
+        if (!file.exists() || !file.canWrite()) {
+            throw new RuntimeException("Such file does not exist: " + fromFileName);
```

Comment:
The error message 'Such file does not exist' is misleading because the condition also checks if the file is writable, which is not relevant here. Consider changing the message to 'Cannot read from file'.